packages/plugin-audio/web/src/modals/AudioModal.tsx

```diff
@@ -10,8 +10,8 @@ const AudioModal = props => {
     experiments,
   } = props;
 
-  const { uploadService, updateService }: UploadContextType = experiments?.useUploadContext?.enabled
-    ? useContext(UploadServiceContext)
+  const { uploadService, updateService }: UploadContextType = experiments?.tiptapEditor?.enabled
+    ? useContext(UploadContext)
```

Comment:
```suggestion
  const { uploadService, updateService } =  useContext(UploadContext)
```
The condition is useless since this component is only rendered on Tiptap